Component/Avatar/avatar.css

```diff
@@ -1 +1,143 @@
-/* TODO: start making */
\ No newline at end of file
+:root{
+  --primary-color:#7c3aed;
+  --light-color: #a78bfa;
+  --border-color:#a8a29e;
+
+}
+nav {
+  width: 90vw;
+  height: 6rem;
```

Comment:
Didn't go through your structure or any html files. Pardon me if I'm wrong.

You can define these set of css for navigation and its link at a single css file and Import that file across your project rather than writing in everywhere.  

You can import you css file in another css file in the following way

```SCSS
// consider this to be your main.css file
// you can import all the css from your individual component css files in your main.css file as below

import url("./path/to/file.css");
// example
import url("./Component/Avatar/avatar.css") // inside your main css file
```

This way if you just import your main css file inside your `<component>.html` file you get all the css from all other files. Only thing that you might have to take care of in this scenario would be using same classnames

Reply:
done